File: airport_nearest_relevant_airport/v1/get/Node SDK/airport_nearest_relevant_airport.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What relevant airports are there around a specific location?
    const response = await waconvo.referenceData.locations.airports.get({
      longitude: 2.55,
      latitude: 49.0,
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: airport_nearest_relevant_airport/v1/get/Python SDK/airport_nearest_relevant_airport.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    What relevant airports are there around a specific location?
    '''
    response = waconvo.reference_data.locations.airports.get(longitude=49.000, latitude=2.55)
    print(response.data)
except ResponseError as error:
    raise error
